README.md

```diff
@@ -30,6 +30,21 @@ Note: If dns servers specified on the host or via dns flags are unreachable from
 
 - https://docs.docker.com/config/containers/container-networking/#dns-services
 
+### docker.login
+
+Authenticate the system user `root` against a set of custom docker registries.
+The registries and the respectively required credentials are provided through pillar dictionaries of the following form
+
+```
+docker-registries:
+  'https://my.docker-registry.com':
+    username: <USER>
+    password: <PW>
+```
+
+The dictionary can be arbitrarily prefixed.
+As long as the dictionary suffix is `-docker-registries`, those credetnials will be picked up by the state.
```

Comment:
I think this refers to the official `docker.login` module, maybe link to the official docs?

Also, `-docker-registries` suffix doesn't match the example, where it's just `docker-registries` (without the prepended `-`)?

Reply:
We should also add a working example to `pillars.example`.

Reply:
cleaned up documentation (linked to main module docs and moved examples for both plain and suffixed dictionaries into `pillar.example`